seastar/src/spawn.hh

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include "cxx-async/include/rust/cxx_async_seastar.h"
+#include "rust/cxx.h"
+#include "cxx_async_futures.hh"
+
+namespace seastar_ffi {
+namespace spawn {
+
+VoidFuture cpp_spawn(VoidFuture future);
+
+}
```

Comment:
Nit: no `// namespace <namespace name>` comments (it's a good practice to have them)